recipes/stingray/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "stingray" %}
+{% set version = "0.1rc1.post2" %}
+{% set sha256 = "f3d17782118427203cf2bbc7ab3cb6625fb1f031b11bd2ec9924418167bb9bf2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - astropy
+    - scipy >=0.18.0
+    - numpy >=1.11.0
+    - matplotlib >=1.3
+  run:
+    - python
+    - astropy
+    - scipy >=0.18.0
+    - numpy >=1.11.0
+    - matplotlib >=1.3
+
+test:
+  imports:
+    - scipy
+    - numpy
+    - matplotlib
+    - astropy
+    - stingray
+    - stingray.modeling
+    - stingray.modeling.tests
+    - stingray.pulse
+    - stingray.pulse.tests
+    - stingray.simulator
+    - stingray.simulator.tests
+    - stingray.tests
+
+about:
+  home: http://github.com/StingraySoftware/stingray
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'The Next Generation Spectral Timing Software'
+
+  description: |
+    This Astropy-affiliated software package merges existing efforts for a timing package in Python
+    and provides the basis for developing spectral-timing analysis tools. It is
+    structured with the best guidelines for modern open-source programming,
+    following the example of Astropy.
+  doc_url: http://stingray.readthedocs.io/
+  dev_url: https://github.com/StingraySoftware/stingray
+
+extra:
+  recipe-maintainers:
+    - matteobachetti
+    - dhuppenkothen
```

Comment:
I know @dhuppenkothen is involved in stingray, but it would be nice to have her confirm willingness to be maintainer. 

This should **not** block merging though...

Reply:
Not knowing anything about conda, I'm not sure how useful I'll be, but I'll confirm anyway that I'm happy to be a maintainer. :)
Thanks for checking!